avaframe/tests/test_com2AB.py

```diff
@@ -108,6 +108,24 @@ def test_calcAB(capfd):
             alphaSD[2] == pytest.approx(alphaSDref[2], rel=tol))
 
 
+def test_writeABtoSHP(capfd):
+    '''test writing to shapefile'''
+
+    avaName = 'avaSlide'
+    dirname = pathlib.Path(__file__).parents[0]
+    avalancheDir = dirname / '..' / 'data' / avaName
+    cfg = cfgUtils.getModuleConfig(com2AB)
+
+    # run main routine
+    resAB = com2AB.com2ABMain(cfg, avalancheDir)
+    abShpFile = outAB.writeABtoSHP(resAB)
```

Comment:
I think Anna said we should first copy the inputs to a temporary path, run and save the results there so that it is then automatically removed at the end of the test.

Reply:
ah, totally forgot! Thanks, will fix